Misc/NEWS.d/next/Library/2024-07-03-14-23-04.gh-issue-119004.L5MoUu.rst

```diff
@@ -0,0 +1,2 @@
+Fix a crash in :ref:`OrderedDict.__eq__ <collections_OrderedDict__eq__>`
+when operands are mutated during the check.  Patch by Bénédikt Tran.
```

Comment:
```suggestion
when operands are mutated during the check. Patch by Bénédikt Tran.
```

Reply:
This is not wrong.

Reply:
> This is not wrong.

I've checked other `NEWS` entries and there's a mix of two or one space after the comma. I'm don't know why this is not wrong, so feel free to ignore my comment.

Reply:
It's common to use 2 spaces after a period just for reading purposes. Historically, this is because of typewriters where it could be hard for human eyes to spot the end of a sentence. Nowawdays, this can be true for some fonts, especially monospace ones where everything has the same width. 

I personally don't use 2-spaces so thanks for spotting it (though I'm not consistent all the time and I'm pretty sure I've committed rst content with 2 spaces after a period).